### PR TITLE
util: GetByteSizeString() take precision argument

### PR DESCRIPTION
**Pull request by brauner**
Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>

### CODE REVIEW

File: lxc/image.go [hunk 1]
```diff
@@ -425,7 +425,7 @@ func (c *imageCmd) run(config *lxd.Config, args []string) error {
 		} else {
 			progress := ProgressRenderer{Format: i18n.G("Transferring image: %s")}
 			handler := func(percent int64, speed int64) {
-				progress.Update(fmt.Sprintf("%d%% (%s/s)", percent, shared.GetByteSizeString(speed)))
+				progress.Update(fmt.Sprintf("%d%% (%s/s)", percent, shared.GetByteSizeString(speed, -1)))
```
Comment (tych0) [inline lxc/image.go]: I know this is minor, but is there any reason to pass -1 here and correct it in the function? Can we just make the argument a uint and pass 2 everywhere?

Comment (brauner) [inline lxc/image.go]: Sure. I thought that this way we simply enforce a default.

Comment (stgraber) [inline lxc/image.go]: I agree with @tych0 here. It's an internal function so might as well make things explicit and just pass the precision everywhere.

**Comment by brauner** [comment]
Updated. Btw, I'd like to have this change for the storage stuff.

**Pull request merged**